includes/widgets/tabs.php

```diff
@@ -427,12 +428,17 @@ protected function render() {
 						'data-tab' => $tab_count,
 						'role' => 'tabpanel',
 						'aria-labelledby' => 'elementor-tab-title-' . $id_int . $tab_count,
+						'tabindex' => '0',
```

Comment:
Are all the tabs `tabindex` suppose to be 0? Including the first one?

Reply:
Yes. They're all hidden and not displayed.
Tabindex doesn't matter unless the element is visible.